server/src/fixed-asset/asset.module.ts

```diff
@@ -3,12 +3,18 @@ import { MongooseModule } from '@nestjs/mongoose';
 import { AssetService } from './asset.service';
 import { AssetController } from './asset.controller';
 import { Asset, AssetSchema } from '../schemas/asset.schema';
+import { User, UserSchema } from '../schemas/user.schema';
 import { AuthGuard } from '../auth/auth.guard';
 import { RolesGuard } from '../auth/roles.guard';
 import { APP_GUARD } from '@nestjs/core';
 
 @Module({
-  imports: [MongooseModule.forFeature([{ name: Asset.name, schema: AssetSchema }])],
+  imports: [
+    MongooseModule.forFeature([
+      { name: Asset.name, schema: AssetSchema },
+      { name: User.name, schema: UserSchema },
```

Comment:
Can you explain why the User model is used here instead of injecting UserService for example?